packages/core/package.json

```diff
@@ -30,29 +31,23 @@
     "prepublishOnly": "npm run build",
     "clean": "npm run clean-module",
     "clean-module": "rimraf ./dist",
-    "build": "tsc -p tsconfig.json",
+    "build": "rollup --config rollup.config.mjs",
```

Comment:
ooohh! aaahh! You've done rollup here too! Nice!!

Reply:
It's just for the core though, I think the browser relies on WebPack a bit more so that will be more of a pickle. Here is allows to output ESM as well as CJS (which I don't think was actually needed in retrospective ^^)